typings/index.d.ts

```diff
@@ -184,6 +184,7 @@ export abstract class AnonymousGuild extends BaseGuild {
   public banner: string | null;
   public description: string | null;
   public nsfwLevel: NSFWLevel;
+  public premiumSubscriptionCount: number | null;
```

Comment:
Wait a minute...isn't it optional or number :thonk:

Reply:
It's [documented as potentially missing](https://discord.com/developers/docs/resources/guild#guild-object) so it looks like there will be a case where this will be `undefined`. Seems like we should handle that.